gobblin-runtime/src/main/java/gobblin/runtime/mapreduce/MRJobLauncher.java

```diff
@@ -594,7 +594,9 @@ protected void cleanup(Context context) throws IOException, InterruptedException
         // Ignored
       } finally {
         if (this.jobMetrics.isPresent()) {
+          this.jobMetrics.get().triggerMetricReporting();
```

Comment:
Better to switch the order between the two so scheduled reporting won't happen while manual reporting is in progress.